platform/android/export/export_plugin.cpp

```diff
@@ -57,6 +57,10 @@
 #include "modules/svg/image_loader_svg.h"
 #endif
 
+#ifdef ANDROID_ENABLED
+#include "platform/android/os_android.h"
```

Comment:
Platform includes should be relative:
```suggestion
#include "../os_android.h"
```